packages/aria2.rb

```diff
@@ -47,6 +40,6 @@ def self.build
 
   def self.install
     system 'make', "DESTDIR=#{CREW_DEST_DIR}", 'install'
-    FileUtils.ln_s "#{CREW_PREFIX}/bin/#{ARCH}-cros-linux-#{@gnuabi}-aria2c", "#{CREW_DEST_PREFIX}/bin/aria2c"
+    FileUtils.ln_s "#{CREW_PREFIX}/bin/#{CREW_BUILD}-aria2c", "#{CREW_DEST_PREFIX}/bin/aria2c"
```

Comment:
Do we need new binaries?

Reply:
Nope. This is is just a simplification of the package file, with no changes to binaries.

Reply:
Great, just wanted to verify.